### ISSUE TITLE
Marker numbering digit sizes do not fit inside triangular marker labels.

### ISSUE THREAD

**Issue by YuryW** [jankae/LibreVNA#148]
### Summary

LibreVNA Version (64 bit): 1.4.0-7f6b78992
OS: Windows 7 Version 6.1 (Build 7601)
CPU Arch: x86_64

Marker numbers on charts do not fit in their triangular labels. Is it possible to make, like other VNAs, the marker number above the triangular mark?
![Marker - копия](https://user-images.githubusercontent.com/79474615/186719157-7e9958d7-326f-4a21-9be9-5acf3c4470a1.jpg)

### Paths to implement it

And it would also be chic, probably, if the user could determine the position of the number in relation to the marker icon (triangle) himself in the menu bar of markers (lower right corner of the application) to the right of the data - numbers above (ab), below (be), to the left (le ) or to the right (ri) of the marker label (or no abbreviations).

![Marker_1 - копия](https://user-images.githubusercontent.com/79474615/186719240-0d3c04aa-442f-4777-aae1-dbcb717cc408.jpg)

### Extra information

An example of placing the marker number above the marker label in one of the VNAs.
![Marker_2 - копия](https://user-images.githubusercontent.com/79474615/186722099-bd150d40-51c7-4b9d-90bf-dc5f9708a67c.jpg)

**Comment by jankae** [comment]
Thank you for the suggestion, the marker number are indeed very poorly readable. I have added a selection of different marker styles in the preferences, here is an example of how the markers can look now:
![markers](https://user-images.githubusercontent.com/16818866/188315657-62c8894f-d5af-4354-8db2-ad11745d6fd4.png)

I did not add on option to display the number anywhere else than above the triangle. My opinion is that this is sufficient for almost all use cases and adding this option would introduce complexity to the software: the marker class generates the symbol, but its placement on the graph is done by the individual graph classes. With the number beside the triangle, the position at which the overall marker icon (number + triangle is drawn) would have to be adjusted accordingly, requiring additional information exchange between these classes. This is certainly doable and I am happy to accept PRs for this if anymore deems this feature absolutely necessary.
